lib/src/async/stream_buffer.dart

```diff
@@ -91,7 +91,7 @@ class StreamBuffer<T> implements StreamConsumer {
 
   List<T> _consume(int size) {
     var follower = 0;
-    var ret = new List(size);
+    final ret = new List<T>(size);
```

Comment:
I'm a fan of `final` myself, but to stay consistent with the rest of the code let's continue using `var` unless strong mode requires something else.